### ISSUE TITLE
"Set cover art as the folder's icon" for already existing library folders?

### ISSUE THREAD

**Issue by lexia1317** [rmcrackan/Libation#815]
Hello, 
I wonder if there is a way to trigger "Set cover art as the folder's icon" for existing library without deleting and re-downloading all the books completely? 
(For situations where a change in folder view/customization settings on Windows 10 caused the custom icons to be overwritten by default folder icons, for example.) 

Also, am I correct in the assumption, that the cover art icon files generated by Libation are currently not stored anywhere? While searching through Libation folders I was only able to find .jpg files, but no .ico files.

Thank you!

**Comment by rmcrackan** [comment]
Good idea. I'd never considered doing it after the fact.

> no .ico files

During this step, Libation grabs the 300x300 pixel version from that images folder. On the fly it turns the image into a hidden `Icon.ico` file and uses that.

**Comment by Mycologist** [comment]
I'd also be quite interested in this feature. Libation only set icons on 3 of my downloaded books.

**Comment by madstan69** [comment]
Yeah, I have books that it's unable to download pics/icons for. So, to add an option to be able to select pics would be awesome as I tried converting to similar size/pixel of icon and it's not detected even after adding the desktop.ini file like other folders have.

<img width="280" height="99" alt="Image" src="https://github.com/user-attachments/assets/18c3eeb5-a076-425d-be0c-5805176f79c3" />

**Comment by rmcrackan** [comment]
This is a fine request but after all this time, clearly I'm not actually going to get around to it. I think it's time for me to just admit defeat.
